#!/usr/bin/env python
PACKAGE = "turtlebot3_autorace_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("top_x",        int_t,      0,      "Top X Pos",        60,  0, 120)
gen.add("top_y",        int_t,      0,      "Top Y Pos",        50,  0, 120)
gen.add("bottom_x",     int_t,      0,      "Bottom X Pos",     140,  0, 320)
gen.add("bottom_y",     int_t,      0,      "Bottom Y Pos",     120,  0, 320)

exit(gen.generate(PACKAGE, "turtlebot3_autorace_camera", "ImageProjectionParams"))